packages/js/src/Modules/Verto/messages/WebRTCStats.ts

```diff
@@ -3,23 +3,25 @@ import BaseMessage from './BaseMessage';
 const DEBUG_REPORT_VERSION = 1;
 
 export class DebugReportStartMessage extends BaseMessage {
-  constructor(id: string) {
+  constructor(id: string, callID: string) {
     super();
     this.buildRequest({
       type: 'debug_report_start',
       debug_report_id: id,
       debug_report_version: DEBUG_REPORT_VERSION,
+      call_id: callID,
```

Comment:
the proxy will pick this callId in `debug_report_start` and `debug_report_stop` messages after changes in vsp, right